scania-digital-design-system/tegel
# fix(storybook): locking dependencies

## **Describe pull-request**  
There is some update, in some of the dependencies that break the build of a storybook.
I was inspecting to see which one exactly, but did not manage to isolate one, so what I did was to "lock" all dependencies. We should consider moving to a newer version of Storybook as we are on version 6 and version 8 is announced already. Probably we would have more issues like this soon if we do not migrate. 

## **Issue Linking:**  
- **No issue:** Issue that was discovered in Table work, not reported.

## **How to test**  
1. Open current develop branch.
2. Go to packages/core
3. Drop node_modules and package-lock.json file
4. Try to install
5. Try to build
6. Try to run Storybook
7. Probably you will encounter warnings and error
8. Try the same with this branch, it should work.

## **Checklist before submission**
